### PR TITLE
Glossary updates - Daml

### PR DESCRIPTION
**Pull request by carrielaben-da**
Daml Concepts through User Management

### CODE REVIEW

File: docs/2.7.4/docs/concepts/glossary.rst [hunk 1]
```diff
@@ -2,491 +2,387 @@
...
-----------------------------------------
+A purpose-built language for the rapid development of `composable <#composability>`__ multi-party `applications <#daml-application-app>`__. It is a modern, ergonomically designed functional language that carefully avoids many
+of the pitfalls that hinder multi-party application development in other languages.
 
-A **disjunction choice** has more than one `controller <#controller>`__.
+Daml Ledger
+===========
 
-If a contract uses **flexible controllers**, this means you don't specify the controller of the `choice <#choice>`__ at `creation <#create>`__ time of the `contract <#contract>`__, but at `exercise <#exercise>`__ time.
+The union of all Daml `contract <#contract>`__ records stored by the `participant nodes <#participant-node>`__. These records form a virtual global shared ledger in the sense that all contracts and changes to them are synchronized across all participant nodes hosting one of their stakeholder parties.
```
Comment (ray-roestenburg-da) [inline docs/2.7.4/docs/concepts/glossary.rst:64]:
```suggestion
The union of all Daml `contract <#contract>`__ records stored by the `participant nodes <#participant-node>`__. These records form a virtual global shared ledger in the sense that all contracts and changes to them are synchronized across all participant nodes hosting at least one of their stakeholder parties.
```

File: docs/2.7.4/docs/concepts/glossary.rst [hunk 2]
```diff
@@ -2,491 +2,387 @@
...
-Use the **command submission service** to `submit commands <#submitting-commands-writing-to-the-ledger>`__ - either create commands or exercise commands - to the `ledger <#daml-ledger>`__. See :ref:`command-submission-service`.
+The following libraries wrap the ledger API for more native experience application development.
 
-Command Completion Service
---------------------------
+Java Bindings
+-------------
 
-Use the **command completion service** to find out whether or not `commands you have submitted <#submitting-commands-writing-to-the-ledger>`__ have completed, and what their status was. See :ref:`command-completion-service`.
+An idiomatic Java library for interacting with the ledger API.JVM of the ledger API. See `Java Bindings </app-dev/bindings-java/index.html>`__.
```
Comment (ray-roestenburg-da) [inline docs/2.7.4/docs/concepts/glossary.rst:266]:
```suggestion
An idiomatic Java library for interacting with the ledger API. See `Java Bindings </app-dev/bindings-java/index.html>`__.
```

**Review by ray-roestenburg-da — APPROVED** [review]
Thanks, only a couple of minor comments for the Glossary.

**Review by loiskilberg-da — APPROVED** [review]
approving for docs team

**Pull request merged**